packages/cxl-ui/scss/cxl-light-card.scss

```diff
@@ -1,68 +1,172 @@
+/* stylelint-disable value-no-vendor-prefix, property-no-vendor-prefix -- some of these are necessary for line-clamp implementation */
+@use "~@conversionxl/cxl-lumo-styles/scss/mq";
+@use "~@conversionxl/cxl-ui/scss/mixins";
+
 :host {
-  min-width: 267px; // 3col widths on 1400px
-  max-width: 300px;
-  height: auto;
-  min-height: calc(3 * var(--lumo-space-xl));
-  padding: var(--lumo-space-m);
-
-  .container > .attributes {
-    display: none;
+  position: relative;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 100%;
+  height: 100%;
+  min-height: 122px;
+  min-width: 267px;
+  padding: var(--lumo-space-m) var(--lumo-space-s) var(--lumo-space-s) var(--lumo-space-m);
+  font-size: var(--lumo-font-size-s);
+  background: var(--lumo-tint);
+  border: 1px solid var(--lumo-contrast-10pct);
+  border-radius: var(--lumo-border-radius-l);
+  box-shadow: var(--lumo-box-shadow-xs);
+  break-inside: avoid;
+  transform: translateZ(0); // CSS columns @see https://stackoverflow.com/a/55110789/35946
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+:host(:hover) {
+  border-color: var(--lumo-primary-color);
+}
+
+:host([portrait]) {
+  display: block;
+  width: 175px;
+  min-width: 0;
+  padding: 0;
+}
+
+:host([completed]) {
+  opacity: 0.6;
+}
+
+:host([theme~="minidegree"]) {
+  background-color: var(--lumo-contrast);
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.badge-new {
+  position: absolute;
+  right: var(--lumo-space-s);
+  bottom: var(--lumo-space-s);
+  display: none;
+  padding: 2px 8px;
+  font-size: 14px;
+  font-weight: 700;
+  color: var(--lumo-primary-contrast-color);
+  border-radius: 40px;
+  background: var(--lumo-primary-color);
+  z-index: 2;
+
+  :host([new]) & {
+    display: block;
   }
+}
 
-  header {
-    .info {
-      .name {
-        font-size: var(--lumo-font-size-m);
-        font-weight: 600;
-      }
+header {
+  display: flex;
+  align-items: start;
+  flex-direction: column;
+  flex-grow: 1;
+  justify-content: space-between;
+  padding-right: 112px; // avatar width
 
-      .attributes {
-        display: flex;
-        flex-direction: column;
-        align-items: flex-start;
-        padding-top: var(--lumo-space-xs);
-        padding-bottom: 0;
-        gap: var(--lumo-space-xs);
-      }
+  :host([portrait]) & {
+    padding: var(--lumo-space-m) 12px 0 12px;
```

Comment:
`--lumo` var should be used for all values. Find closest value or make a calculation eg. by doing 50% of value